packages/types/src/codec/create/getTypeDef.ts

```diff
@@ -59,15 +59,15 @@ function _decodeStruct (value: TypeDef, type: string, subType: string): TypeDef
   value.info = TypeDefInfo.Struct;
   value.sub = keys.map((name): TypeDef =>
     // eslint-disable-next-line @typescript-eslint/no-use-before-define
-    getTypeDef(parsed[name], name)
+    getTypeDef(parsed[name], { name })
   );
 
   return value;
 }
 
 // decode a fixed vector, e.g. [u8;32]
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
-function _decodeFixedVec (value: TypeDef, type: string, subType: string): TypeDef {
+function _decodeFixedVec (value: TypeDef, type: string): TypeDef {
```

Comment:
If the unsued var is removed, the eslint comment can also go. (Would generally prefer to keep it though to keep all the signatures consistent, rather `_` it is not used)